src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -14,6 +14,7 @@ var extendFlat = require('../../../lib/extend').extendFlat;
 
 
 module.exports = {
+    visible: axesAttrs.visible,
```

Comment:
Yeah, correct. Was contemplating the best option as was working back through that. Correct though, it was just to make them *not fail*.